modules/network-observability-viewing-alerts.adoc

```diff
@@ -9,7 +9,9 @@
 You can access metrics about health and resource usage of the Network Observability Operator from the *Dashboards* page in the web console. A health alert banner that directs you to the dashboard can appear on the *Network Traffic* and *Home* pages in the event that an alert is triggered. Alerts are generated in the following cases:
 
 * The *NetObservLokiError* alert occurs if the `flowlogs-pipeline` workload is dropping flows because of Loki errors, such as if the Loki ingestion rate limit has been reached.
-* The *NetObservNoFlows* alert occurs if no flows are ingested for a certain amount of time..Prerequisites
+* The *NetObservNoFlows* alert occurs if no flows are ingested for a certain amount of time.
```

Comment:
This is a very un-related change; it correct a line-spacing error in the docs.

Reply:
- Are these alert names UI elements on the Dashboard page? If so, bold formatting is OK. But if not, then you might want to format the alert names as literals, like we we do in the Monitoring book: `NetObservNoFlows`.
- Also, should we be more specific than `a certain amount of time`?

Reply:
Hm. Yeah these are not UI elements, so your suggestion to format as literals is a good one. Thanks! It is out of scope I guess, but technically this prerequisite line amendment is too. I don't have an issue with fixing the formatting here.

Reply:
Addressed.